ros/api/v0/hosts.py

```diff
@@ -186,10 +240,20 @@ def get(self, host_id):
             record = {'inventory_id': host_id}
             record['display_performance_score'] = profile.display_performance_score
             record['rating'] = rating_record.rating if rating_record else None
-            record['recommendation_count'] = len(system.rule_hit_details)
-            record['state'] = system.rule_hit_details[0].get('key')
+            record['report_date'] = profile.report_date
+            record['cloud_provider'] = system.cloud_provider
+            record['instance_type'] = system.instance_type
+            record['idling_time'] = profile.idling_time
+            record['io_wait'] = profile.io_wait
+            record['rule_hit_details'] = system.rule_hit_details
```

Comment:
Hello @apuntamb, @patilsuraj767 
As discussed, we might need a separate API for recommendations. In that case, will it be good to skip field `rule_hits_detail` here as well?